webui/react/src/components/NotesCard.tsx

```diff
@@ -135,7 +135,7 @@ const NotesCard: React.FC<Props> = (
           onClick={handleNotesClick}
         />
       </Spinner>
-      <Prompt
+      {/* <Prompt
```

Comment:
https://github.com/remix-run/react-router/issues/8139
`Prompt` is not available in v6